lib/index.html

```diff
@@ -19,6 +28,13 @@
 
       function execKarmaMethod(evt) {
         var msg = evt.data;
+        if (msg.command === 'result') {
```

Comment:
As you can see later, there is another check to see if command is `complete`. Can you refactor this to use a map of commands and actions, please?